src/ensembl/tests/fixtures/entity-viewer/gene.ts

```diff
@@ -40,7 +46,7 @@ export const createGene = (fragment: Partial<FullGene> = {}): FullGene => {
     symbol: faker.lorem.word(),
     name: faker.lorem.words(),
     slice: geneSlice,
-    transcripts: [transcript],
+    transcripts: transcripts,
```

Comment:
No need to specify value separately when both the key and value pair are the same